src/utils.js

```diff
@@ -12,3 +13,21 @@ export function getDataHubParams(partialUrl) {
     search: parsedUrl.query,
   };
 }
+
+export function useInterval(callback, delay) {
```

Comment:
Do you think that could be possible testing this hook?

Maybe there is some useful things here:
* https://github.com/testing-library/react-hooks-testing-library
* https://medium.com/@nitinpatel_20236/unit-testing-custom-react-hooks-caa86f58510

Besides the problem of how testing the hooks, we have another problem here, how to test when _setInterval_ is involved. Maybe this could help on this: https://jestjs.io/docs/en/timer-mocks

Reply:
Implemented.